highfidelity/hifi
# Case 20322, Case 12279, Case 15803: Improve PolyLines to support glow and faceCamera

https://highfidelity.manuscript.com/f/cases/20322/Modify-PolyLines-to-support-everything-Line3D-Overlays-can-do
https://highfidelity.manuscript.com/f/cases/12279/PolyLine-s-registrationPoint-value-is-wrong
https://highfidelity.manuscript.com/f/cases/15803/Crash-while-rendering-PolyLineEntity

New PolyLine properties and re-working how we render polylines to be more efficient.

Test plan:
- run the content/entity/polyline tests from this branch: https://github.com/highfidelity/hifi_tests/tree/f92b377c5a36a484f7a8f601fd9acf2a181e68f6/tests/content/entity/polyline
- verify that the Fingerpaint (beta) app in the marketplace still works as in master
- Open Create and select one of the PolyLines you've created.  Change its registration point.  The line/bounding box should move as expected.
